arangod/Aql/Executor/JoinExecutor.cpp

```diff
@@ -60,7 +61,7 @@ void JoinExecutorInfos::determineProjectionsForRegisters() {
 }
 
 JoinExecutor::~JoinExecutor() {
-  clearProjectionsBuilder();
+  _projectionsBuilder.clear();
```

Comment:
Can clearProjectionsBuilder() be deleted from the .h as it's not used anymore?

Reply:
Yes please remove function form header and implementation please.
The implementation now is actually wrong (miscounting) so we need to make sure we cannot accidentally reintroduce it.